From 063c0ad450d641a4ffc1b90dd8053f799e0ef813 Mon Sep 17 00:00:00 2001
From: dhacker29 <dhackerdvm@gmail.com>
Date: Sat, 25 Jan 2014 20:17:13 -0600
Subject: [PATCH 09/18] Add date to package name

---
 core/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/Makefile b/core/Makefile
index 34dd568..020137b 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1473,7 +1473,8 @@ name := $(TARGET_PRODUCT)
 ifeq ($(TARGET_BUILD_TYPE),debug)
   name := $(name)_debug
 endif
-name := $(name)-ota-$(FILE_NAME_TAG)
+date := $(shell date -u +%Y%m%d)
+name := $(name)-ota-$(FILE_NAME_TAG)-$(date)
 
 INTERNAL_OTA_PACKAGE_TARGET := $(PRODUCT_OUT)/$(name).zip
 
-- 
2.1.0

